development/kops/eks-d.tpl

```diff
@@ -24,6 +24,13 @@ spec:
       name: a
     memoryRequest: 100Mi
     name: events
+  externalPolicies:
```

Comment:
Is this going to prevent customers from creating a kops cluster with these scripts. I thought we had testing specific stuff isolated out ouf the default kops creat scripts.

Reply:
No, this should work fine. This policy is the minimum required for SSM agent to operate, it is _not_ administrative SSM permissions.